src/Components/CaseStatusBar/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../styles/colors';
+export default StyleSheet.create({
+  caseStatusContainer: {
+    marginTop: 20,
+    aspectRatio: 7.4,
```

Comment:
Let's use `width` and `height` instead of `aspectRatio` for consistency. We likely have more control this way as well. Try to keep styling consistent with the other components on the `CaseScreen` such as the `EligibilityCard` and `CaseSummaryCard`

Reply:
Resolving for now because we can worry about styling later.